integration_tests/commands/getex_test.go

```diff
@@ -119,6 +119,30 @@ func TestGetEx(t *testing.T) {
 			assert_type: []string{"equal", "equal", "assert", "equal", "assert"},
 			delay:       []time.Duration{0, 0, 0, 0, 0},
 		},
+		{
+			name:        "GetEx with key holding JSON type",
+			commands:    []string{"JSON.SET KEY $ \"1\"", "GETEX KEY"},
+			expected:    []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
+			assert_type: []string{"equal", "equal"},
+			delay:       []time.Duration{0, 0},
+		},
+		{
+			name:     "GetEx with key holding JSON type with multiple set commands",
+			commands: []string{"JSON.SET KEY $ \"{\"a\":2}\"", "GETEX KEY", "JSON.SET KEY $.a \"3\"", "GETEX KEY"},
+			expected: []interface{}{"OK",
+				"WRONGTYPE Operation against a key holding the wrong kind of value",
+				"OK",
+				"WRONGTYPE Operation against a key holding the wrong kind of value"},
+			assert_type: []string{"equal", "equal", "equal", "equal"},
+			delay:       []time.Duration{0, 0, 0, 0},
+		},
+		{
+			name:        "GetEx with key holding SET type",
+			commands:    []string{"SADD KEY 1 2 3", "GETEX KEY"},
+			expected:    []interface{}{"OK", "WRONGTYPE Operation against a key holding the wrong kind of value"},
+			assert_type: []string{"equal", "equal"},
```

Comment:
Hi, the test is failing as you are not deleting key with name `KEY` before executing an `SADD` command. Please add a delete command for `KEY`. please see line 151 for reference.

Reply:
incorporated. I have changed key name. Ideally, tests should be independent.